robot_skills/package.xml

```diff
@@ -47,6 +47,7 @@
   <exec_depend condition="$ROS_PYTHON_VERSION == 3">python3-mock</exec_depend>
 
   <test_depend>test_tools</test_depend>
+  <test_depend>hero_description</test_depend>
```

Comment:
I think this one can be removed now with the seperated skills

Reply:
sharp, I missed that one

Reply:
```suggestion
```